src/components/Preview.js

```diff
@@ -105,6 +105,7 @@ function Preview({ markup, variant, forwardedRef, dispatch }) {
         scrolling="no"
         frameBorder="0"
         onLoad={handleLoadIframe}
+        data-testid="sandbox"
```

Comment:
Same as for the other comment. What du you think if we add a `title` to the `iframe`? Then we can you a `getByTitle`. 


```suggestion
        title="sanbox"
```

Reply:
@marcosvega91 great suggestion!

Reply:
was sandbox not sanbox. Sorry for that xD